### PR TITLE
[conf] Small fixes TU Delft

### PR DESCRIPTION
**Pull request by fvantienen**

### CODE REVIEW

**Comment by gautierhattenberger** [comment]
You have reverted some changes that were "fixing" some errors:
- the replacement of `->` by `@DEREF` in the flight plans
- the `test` node in the nerderdrone control module that is not complete and was removed as you are the only user of this
- some commented codes in `target_pos.c` that were previously removed

**Comment by fvantienen** [comment]
Sorry ,some were to be changed back but some weren't. Don't know why but the `@DEREF` doesn't work so that one had to be reverted.

**Comment by gautierhattenberger** [comment]
It seems that the call and call_once functions are not parsed to replace the statements like DEREF. It is is just a direct copy of the code in the final flight plan. I'm not sure if it is a real choice or just something that we have forgotten.

File: conf/modules/eff_scheduling_nederdrone.xml [hunk 1]
```diff
@@ -26,5 +26,6 @@ not use this module at the same time!</description>
   <periodic fun="ctrl_eff_scheduling_periodic()" freq="20.0"/>
   <makefile target="ap|nps" firmware="rotorcraft">
     <file name="eff_scheduling_nederdrone.c"/>
+    <test/>
```
Comment (gautierhattenberger) [inline conf/modules/eff_scheduling_nederdrone.xml:29]:
```suggestion
```

File: sw/airborne/modules/ctrl/target_pos.c [hunk 2]
```diff
@@ -74,6 +74,9 @@ struct target_t target = {
   .integrate_z = TARGET_INTEGRATE_Z
 };
 
+/* Get the Relative postion from the RTK */
```
Comment (gautierhattenberger) [inline sw/airborne/modules/ctrl/target_pos.c:77]: remove again commented parts ?

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]

File: sw/airborne/modules/ctrl/target_pos.c [hunk 3]
```diff
@@ -74,6 +74,9 @@ struct target_t target = {
...
+//extern struct GpsRelposNED gps_relposned;
+
```
Comment (gautierhattenberger) [inline sw/airborne/modules/ctrl/target_pos.c:79]:
```suggestion
```

File: sw/airborne/modules/ctrl/target_pos.c [hunk 4]
```diff
@@ -142,6 +145,16 @@ void target_parse_target_pos(uint8_t *buf)
...
+  // /* When we have a valid relative position from the RTK GPS and no timeout update the position */
+  // if((gps_relposned.relPosValid != 0) && (gps_relposned.iTOW+target.rtk_timeout) > gps_tow_from_sys_ticks(sys_time.nb_tick)) {
+  //   struct NedCoor_f cur_pos = *stateGetPositionNed_f();
+
+  //   // Convert the relative postion to a postion in the NED frame of the UAV
+  //   pos->x = cur_pos.x - gps_relposned.relPosN / 100.0f;
+  //   pos->y = cur_pos.y - gps_relposned.relPosE / 100.0f;
+  //   pos->z = cur_pos.z - gps_relposned.relPosD / 100.0f;
+  // }
+
```
Comment (gautierhattenberger) [inline sw/airborne/modules/ctrl/target_pos.c:157]:
```suggestion
```

**Review by gautierhattenberger — APPROVED** [review]

**Pull request merged**